### PR TITLE
Add weave-gitops as a helm chart

### PR DESCRIPTION
**Pull request by J-Thompson12**
Closes: #1379

**What changed?**
Added a chart to weave-gitops to allow installing weave-gitops as a helmrelease to existing flux.

**Why?**

**How did you test it?**

**Release notes**

**Documentation Changes**

### CODE REVIEW

File: charts/weave-gitops/templates/wego-app.yaml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+          ports:
+            - containerPort: 9001
+              protocol: TCP
+          imagePullPolicy: IfNotPresent
+  selector:
+    matchLabels:
+      app: wego-app
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
```
Comment (mtparet) [inline charts/weave-gitops/templates/wego-app.yaml:27]: Should we use a ClusterRoleBinding ? When installing the chart we have currently in the logs :
```
E0209 15:49:45.385748       1 reflector.go:138] pkg/mod/k8s.io/client-go@v0.22.2/tools/cache/reflector.go:167: Failed to watch *v1beta1.HelmRepository: failed to list *v1beta1.HelmRepository: helmrepositories.source.toolkit.fluxcd.io is forbidden: User "system:serviceaccount:flux-system:wego-app-service-account" cannot list resource "helmrepositories" in API group "source.toolkit.fluxcd.io" at the cluster scope
```

Comment (mtparet) [reply charts/weave-gitops/templates/wego-app.yaml:27]: With a clusterrolebinding it is working.
I made a PR for change suggestions https://github.com/weaveworks/weave-gitops/pull/1422

Comment (J-Thompson12) [reply charts/weave-gitops/templates/wego-app.yaml:27]: @mtparet This a proof of concept idea but I did put up this draft pr and ask team members why I was getting this error. So thank you for helping out with that. Most likely the chart will not actually sit in this repo.

**Comment by Callisto13** [comment]
@J-Thompson12 are you actively taking this forwards? Or is this as far as you are going with this POC?

**Pull request closed without merging**